app/components/emoji/emoji.js

```diff
@@ -157,6 +145,29 @@ export default class Emoji extends React.PureComponent {
         // force a new image to be rendered when the size changes
         const key = Platform.OS === 'android' ? (height + '-' + width) : null;
 
+        if (!imageUrl) {
+            let BlankComponent;
+            if (Platform.OS === 'android') {
```

Comment:
Is this easier if we just use an Image for both platforms? I don't know if using a View on iOS makes much of a difference

Reply:
Felt weird to me use an Image component when there was no image to display, let me try it with an Image component on iOS and if that works fine then I'll just do that instead